data/pages.yaml

```diff
@@ -86,21 +86,22 @@ developer:
   intro: Extend automation with custom Ansible modules, add functionality to existing modules, or fix bugs to improve existing code.
   milestone:
     start: Start writing code
-    contribute: Contribute code to an Ansible project
+    contribute: Contribute code to a collection
     test: Test plugins and modules
     create_collection: Create new collections
   start:
     intro: Ready to start writing code for Ansible? Follow these steps to get started.
     setup: Set up your development environment
-    conduct: Review the Ansible contributor guidelines
```

Comment:
If we remove fields from `data/*.yaml` we need to make sure the change is reflected in the corresponding `template`: https://github.com/ansible/jinja-docsite/blob/7d434fe81020602ee880ffc5d2b247f60c455955/templates/developer.html#L18

Reply:
Aha! Now clear, thanks, changed, PTAL